FULL_HELP_DOCS.md

```diff
@@ -1283,6 +1285,24 @@ Simulate a transaction envelope from stdin
 
 
 
+## `stellar tx hash`
+
+Calculate the hash of a transaction envelope from stdin and prints it as a hex string
+
+**Usage:** `stellar tx hash [OPTIONS] --source-account <SOURCE_ACCOUNT>`
+
+###### **Options:**
+
+* `--rpc-url <RPC_URL>` — RPC server endpoint
+* `--network-passphrase <NETWORK_PASSPHRASE>` — Network passphrase to sign the transaction sent to the rpc server
+* `--network <NETWORK>` — Name of network to use from config
+* `--source-account <SOURCE_ACCOUNT>` — Account that signs the final transaction. Alias `source`. Can be an identity (--source alice), a secret key (--source SC36…), or a seed phrase (--source "kite urban…")
+* `--hd-path <HD_PATH>` — If using a seed phrase, which hierarchical deterministic path to use, e.g. `m/44'/148'/{hd_path}`. Example: `--hd-path 1`. Default: `0`
```

Comment:
@BlaineHeffron @willemneal @fnando The rpc url, source account, and hd path shouldn't show up to the user as options and the command shouldn't show the source account being required. The only options that matter for this command are `--network-passphrase` and `--network` for selecting what the network passphrase is for calculating the hash.

I assume they're showing up because of the way we have organised the network args as a single group that are all always required.

I don't think this issue needs to block merging this PR specifically, but we should follow up with telling a clearer story for these commands.

Reply:
There is `commands::network::Args` which should include just the network info required!